app/helpers/tags_helper.rb

```diff
@@ -76,7 +76,11 @@ def link_to_tag_works(tag, options = {})
   end
 
   def link_to_tag_with_text(tag, link_text, options = {})
-    link_to_with_tag_class(@collection ? collection_tag_path(@collection, tag) : tag_path(tag), link_text, options)
+    if options[:full_path] 
```

Comment:
If we want the full url then we should se the option to be full_path otherwise not :)